File: CarlaChronicles-BackOnTrack/bot3.go
package main

import (
  "strconv"
  "fmt"
  "math"
  //"./mymath"
)

// a_lat max = 12 m/s^2
// v = 30 m/s

func aLat(v, r float64) float64 {
  return math.Pow(v,2) / r
}


func main() {
  var v float64 = 30
  radii := []float64{50, 31.4, 75, 16}

  for _,r := range radii {
    fmt.Println("a_lat for radius " +
      strconv.FormatFloat(r, 'f', -1, 64) +
      ": " +
      strconv.FormatFloat(aLat(v, r), 'f', -1, 64))
  }
}

File: CarlaChronicles-BackOnTrack/bot3.cpp
#include <iostream>
#include <iomanip>
#include <math.h>
#include <vector>
// #include "mymath.cpp" // Tau not pi


static float a_lat(float v, float r)
{
  return pow(v,2.0f) / r;
}

int main()
{
  float v = 30;
  std::vector<float> radii = {50, 31.4f, 75, 16};
  for (auto r : radii) {
    std::cout << "a_lat for radius " << r << ": ";
    std::cout << a_lat(v, r) << "\n";
  }
}
